src/main/java/de/unistuttgart/crosswordbackend/data/Configuration.java

```diff
@@ -39,4 +40,17 @@ public void addQuestion(final Question question) {
     public void removeQuestion(final Question question) {
         this.questions.remove(question);
     }
+
+    @Override
+    public Object clone() {
```

Comment:
By the way, due to covariant return types you can return `Configuration` directly.
Same for any other `clone` override (in all other PRs and this one) as well.

Reply:
By that I simply mean
```suggestion
    public Configuration clone() {
```
which will also work for all other methods of the same schemata.